config/config.go

```diff
@@ -53,6 +56,22 @@ func (c ConnectionType) Start(conf interface{}, enableDebug bool) {
 		}
 		log.Println("Starting ", KafkaFoxtrot)
 		connObj.Run()
+	case KafkaHTTPSideline:
```

Comment:
Do we need separate kind here? can we not add sideline as a capability within http ?

Reply:
If we do it this way then there will be multiple if/else conditions will be introduced in the code which might cause unnecessary confusion. Hence to avoid thought it is better to use it as a new connectionType which keeps code more cleaner.